ci/unit/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG ELASTIC_STACK_VERSION
+FROM docker.elastic.co/logstash/logstash:$ELASTIC_STACK_VERSION
+WORKDIR /usr/share/logstash/logstash-core
+RUN cp versions-gem-copy.yml ../logstash-core-plugin-api/versions-gem-copy.yml
+COPY --chown=logstash:logstash . /usr/share/plugins/this
+WORKDIR /usr/share/plugins/this
+ENV PATH=/usr/share/logstash/vendor/jruby/bin:${PATH}
+ENV LOGSTASH_SOURCE 1
+RUN jruby -S gem install bundler
+RUN jruby -S bundle install --jobs=3 --retry=3
+RUN jruby -S bundle exec rake vendor
```

Comment:
shouldn't we be using Gradle ? also why is this needed here?

Reply:
This is copied from @jsvd work after the jruby S3 bucket went AWOL on the weekend.
Behind the scenes it does use gradle so I suppose we could run gradle here.
I don't want to touch it now, it works, we can circle back after confirming with Joao about this. It is a pattern we need to apply to all (most) plugins - probably scripted.
```
Step 11/11 : RUN jruby -S bundle exec rake vendor
 ---> Running in a4c0913880da
Downloading https://services.gradle.org/distributions/gradle-4.5.1-bin.zip
.....................................................................
Unzipping /usr/share/logstash/.gradle/wrapper/dists/gradle-4.5.1-bin/a5vbgfvpwtoqz8v2cdivxz28k/gradle-4.5.1-bin.zip to /usr/share/logstash/.gradle/wrapper/dists/gradle-4.5.1-bin/a5vbgfvpwtoqz8v2cdivxz28k
Set executable permissions for: /usr/share/logstash/.gradle/wrapper/dists/gradle-4.5.1-bin/a5vbgfvpwtoqz8v2cdivxz28k/gradle-4.5.1/bin/gradle
Starting a Gradle Daemon (subsequent builds will be faster)
:cleanGemjar UP-TO-DATE
:clean UP-TO-DATE
:compileJava
Download https://repo.maven.apache.org/maven2/org/jruby/jruby-complete/9.1.13.0/jruby-complete-9.1.13.0.pom
Download https://repo.maven.apache.org/maven2/org/jruby/jruby-artifacts/9.1.13.0/jruby-artifacts-9.1.13.0.pom
Download https://repo.maven.apache.org/maven2/org/jruby/jruby-parent/9.1.13.0/jruby-parent-9.1.13.0.pom
Download https://repo.maven.apache.org/maven2/org/sonatype/oss/oss-parent/7/oss-parent-7.pom
Download https://repo.maven.apache.org/maven2/org/jruby/jruby-complete/9.1.13.0/jruby-complete-9.1.13.0.jar
:processResources NO-SOURCE
:classes
:jar
:sourcesJar
:copyGemjar
```

Reply:
ok I see.
I just created elastic/logstash/issues/9823 - we can move that discussion over there  (or open a specific issue about it and link it there)